<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalculatorClass</class>
 <widget class="QMainWindow" name="CalculatorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btn5">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>211</y>
      <width>84</width>
      <height>43</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;
font: 14pt &quot;Kristen ITC&quot;;</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDivides">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>251</y>
      <width>84</width>
      <height>43</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;
font: 14pt &quot;Kristen ITC&quot;;</string>
    </property>
    <property name="text">
     <string>รท</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMinus">
    <property name="geometry">
     <rect>
      <x>244</x>
      <y>251</y>
      <width>84</width>
      <height>43</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;
font: 14pt &quot;Kristen ITC&quot;;</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn4">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>211</y>
      <width>84</width>
      <height>43</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;
font: 14pt &quot;Kristen ITC&quot;;</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn6">
    <property name="geometry">
     <rect>
      <x>163</x>
      <y>211</y>
      <width>84</width>
      <height>43</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;
font: 14pt &quot;Kristen ITC&quot;;</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReciprocal">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>171</y>
      <width>84</width>
      <height>43</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;
font: 14pt &quot;Kristen ITC&quot;;</string>
    </property>
    <property name="text">
     <string>1/x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCE">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>166</width>
      <height>43</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;
font: 14pt &quot;Kristen ITC&quot;;</string>
    </property>
    <property name="text">
     <string>CE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn9">
    <property name="geometry">
     <rect>
      <x>163</x>
      <y>171</y>
      <width>84</width>
      <height>43</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;
font: 14pt &quot;Kristen ITC&quot;;</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn1">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>251</y>
      <width>84</width>
      <height>43</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;
font: 14pt &quot;Kristen ITC&quot;;</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNegate">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>291</y>
      <width>84</width>
      <height>43</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;
font: 14pt &quot;Kristen ITC&quot;;</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn3">
    <property name="geometry">
     <rect>
      <x>163</x>
      <y>251</y>
      <width>84</width>
      <height>43</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;
font: 14pt &quot;Kristen ITC&quot;;</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnC">
    <property name="geometry">
     <rect>
      <x>163</x>
      <y>130</y>
      <width>84</width>
      <height>43</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;
font: 14pt &quot;Kristen ITC&quot;;</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnResult">
    <property name="geometry">
     <rect>
      <x>244</x>
      <y>291</y>
      <width>166</width>
      <height>43</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;
font: 14pt &quot;Kristen ITC&quot;;</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn7">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>171</y>
      <width>84</width>
      <height>43</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;
font: 14pt &quot;Kristen ITC&quot;;</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>409</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>16</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn2">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>251</y>
      <width>84</width>
      <height>43</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;
font: 14pt &quot;Kristen ITC&quot;;</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPoint">
    <property name="geometry">
     <rect>
      <x>163</x>
      <y>291</y>
      <width>84</width>
      <height>43</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;
font: 14pt &quot;Kristen ITC&quot;;</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPlus">
    <property name="geometry">
     <rect>
      <x>244</x>
      <y>211</y>
      <width>84</width>
      <height>43</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;
font: 14pt &quot;Kristen ITC&quot;;</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMultiplies">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>211</y>
      <width>84</width>
      <height>43</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;
font: 14pt &quot;Kristen ITC&quot;;</string>
    </property>
    <property name="text">
     <string>ร</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn0">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>291</y>
      <width>84</width>
      <height>43</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;
font: 14pt &quot;Kristen ITC&quot;;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn8">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>171</y>
      <width>84</width>
      <height>43</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;
font: 14pt &quot;Kristen ITC&quot;;</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPercent">
    <property name="geometry">
     <rect>
      <x>244</x>
      <y>171</y>
      <width>84</width>
      <height>43</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;
font: 14pt &quot;Kristen ITC&quot;;</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBack">
    <property name="geometry">
     <rect>
      <x>244</x>
      <y>130</y>
      <width>166</width>
      <height>43</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;
font: 14pt &quot;Kristen ITC&quot;;</string>
    </property>
    <property name="text">
     <string>โ</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>410</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>ๆฅ็</string>
    </property>
    <addaction name="actiona_2"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>็ผ่พ</string>
    </property>
    <addaction name="actiona_3"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>ๅธฎๅฉ</string>
    </property>
    <addaction name="actiona"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actiona">
   <property name="text">
    <string>่ฟ้ฝไธไผ?</string>
   </property>
  </action>
  <action name="actiona_2">
   <property name="text">
    <string>่ฟ้ฝไธไผ?</string>
   </property>
  </action>
  <action name="actiona_3">
   <property name="text">
    <string>่ฟ้ฝไธไผ?</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Calculator.qrc"/>
 </resources>
 <connections/>
</ui>
